sw/device/lib/dif/dif_sensor_ctrl.h

```diff
@@ -0,0 +1,100 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
```

Comment:
@a-will can you take a quick look and see if this is headed in the right direction?

Reply:
This is looking good to me. Note that the style guide would probably want you to spell out `clear` instead of `clr` in the function name.